core/src/main/scala/chisel3/Data.scala

```diff
@@ -680,7 +680,7 @@ abstract class Data extends HasId with NamedComponent with SourceInfoDoc {
           if (mod.flatMap(Builder.retrieveParent(_, Builder.currentModule.get)) == Builder.currentModule) =>
       case Some(_: UnconstrainedBinding) =>
       case _ =>
-        throwException(s"operand '$this' is not visible from the current module")
+        throwException(s"operand '$this' is not visible from the current module: (topBinding = ${topBindingOpt})")
```

Comment:
While I grant that `blah is not visible` is not necessary the best user error message, I'm not sure that introducing the user to binding is quite right either. Is this a change you're intending to include?

Reply:
i mean, i found this useful in debugging the issue, but happy to take any other suggestions.